legal_references.tsv

```diff
@@ -690,11 +691,11 @@ lov om luftfart|luftfartsloven|luftfartsloven	lov/1993-06-11-101
 lov om lønnsnemnd i arbeidstvister	lov/2012-01-27-10
 lov om lønnsplikt under permittering|permitteringslønnsloven|permitteringsloven	lov/1988-05-06-22
 lov om løysingsrettar|løsningsrettsloven|løysingsrettslova|løsningsloven|løysingslova|løysl|løl	lov/1994-12-09-64
-lov om matproduksjon og mattrygghet|matloven	lov/2003-12-19-124
+lov om matproduksjon og mattrygghet|matloven|matl	lov/2003-12-19-124
 lov om matrikkelrevisjonen|matrikkelrevisjonsloven	lov/1863-06-06-1
 lov om medisinsk og helsefaglig forskning|helseforskningsloven|helseforskningloven	lov/2008-06-20-44
-lov om medisinsk utstyr	lov/1995-01-12-6
-lov om megling i konfliktråd|konfliktrådsloven|konfliktrådloven	lov/1991-03-15-3
+lov om medisinsk utstyr|medisinsk utstyrsloven|medutstyrl	lov/1995-01-12-6
+lov om megling i konfliktråd|konfliktrådsloven|konfliktrådloven	lov/2014-06-20-49|lov/1991-03-15-3
```

Comment:
Dette blir feil. 
lov/1991-03-15-3 heter "Lov om megling i konfliktråd", mens lov/2014-06-20-49 heter "Lov om konfliktrådsbehandling".
Altså kan ikke "lov om megling i konfliktråd" mappes til lov/2014-06-20-49.
Det riktige her ville vært
```
konfliktrådsloven|konfliktrådloven	lov/2014-06-20-49|lov/1991-03-15-3
lov om megling i konfliktråd	lov/1991-03-15-3
lov om konfliktrådsbehandling	lov/2014-06-20-49
```

Reply:
Man må være forsiktig med dette når en lov erstatter en annen, for de kan ha samme kortnavn, men forskjellig fullt navn.

Reply:
Jeg har gått igjennom alle de andre linjene her som har fått en ny ID i høyre kolonne, og det virker ikke som dette er et problem med noen av de andre radene; der heter det nye dokumentet akkurat det samme som det gamle.